### ISSUE TITLE
LibreOffice read-only notice unreadable

### ISSUE THREAD

**Issue by pschichtel** [tista500/Adapta#94]
**Reporter info**

 ```
 * Distribution - Archlinux
 * Gtk+ 3.0 version - 3.20.4
 * Desktop environment - GNOME 3.20.1
 ```

**Adapta version (if you knew)**
3.21.1.152

**Related Application and/or Shell theming (or name of widget)**
LibreOffice

**Actual issue**

Read-Only notice with a light font color on a light yellow background
![libreoffice-readonly](https://cloud.githubusercontent.com/assets/526104/15308590/44308a78-1bdf-11e6-833c-24ac9f769b52.png)

**Steps to reproduce (if you knew)**
Open write-protected file with LibreOffice Writer

**Expected behaviour**
Readable text

**Comment by tista500** [comment]
LibreOffice is out of control.

Regards.

**Comment by gregorydk** [comment]
This bug is solved in LibreOffice 5.2 beta.
